.eslintrc.json

```diff
@@ -30,17 +21,8 @@
     "@typescript-eslint/explicit-function-return-type": 0,
     "@typescript-eslint/no-explicit-any": 0,
     "@typescript-eslint/explicit-member-accessibility": 0,
-    "@typescript-eslint/no-unused-vars": [0, {"argsIgnorePattern": "^_"}],
+    "@typescript-eslint/no-unused-vars": [0, { "argsIgnorePattern": "^_" }],
     "no-undefined": 0,
-    "default-case": 0,
-    "import/no-absolute-path": 2,
```

Comment:
Removing all the import ones in favor of letting prettier/eslint take care of this. This _might_ lead to some import reordering in future PRs... but so be it!